.eslintignore

```diff
@@ -1,2 +1,5 @@
 coverage
 lib
+pnpm-lock.yaml
+# Dot files and folders are ignored by default by ESLint
+!.github
```

Comment:
Ah, nice spot. We should also un-exclude `.vscode`. Maybe `!.*` would be a good thing, if it works?

Reply:
Added this in https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/286